distribution/solr-distro/src/main/resources/bin/solr.cmd

```diff
@@ -1328,8 +1328,9 @@ IF "%FG%"=="1" (
 
   REM now wait to see Solr come online ...
   "%JAVA%" %SOLR_SSL_OPTS% %AUTHC_OPTS% %SOLR_ZK_CREDS_AND_ACLS% -Dsolr.install.dir="%SOLR_TIP%" -Dsolr.default.confdir="%DEFAULT_CONFDIR%"^
-    -Dlog4j.configurationFile="%DEFAULT_SERVER_DIR%\scripts\cloud-scripts\log4j2.xml" ^
-    -classpath "%DEFAULT_SERVER_DIR%\solr-webapp\webapp\WEB-INF\lib\*;%DEFAULT_SERVER_DIR%\lib\ext\*" ^
+    -Dlog4j.configurationFile="%DEFAULT_SERVER_DIR%\resources\log4j2.xml" ^
+    -classpath "%DEFAULT_SERVER_DIR%\solr-webapp\webap
+    p\WEB-INF\lib\*;%DEFAULT_SERVER_DIR%\lib\ext\*" ^
```

Comment:
❔ Is this newline okay/intentional?

Reply:
Absolutely not. I must have fat-fingered at some point during the replace-all. I've updated this, and the other PR. Very nice catch!